### ISSUE TITLE
Nested query in rnd_str() does not work

### ISSUE THREAD

**Issue by TheTanc** [questdb/questdb#92]
I am trying to select random instruments from an existing table.

I have a table `instruments` that looks like this

|name          |
|-------------|
|AAPL           |
|ABBV           |
|BAC             |
|...                 |

I am trying to produce a table of `random` instruments as follows:

```sql
SELECT rnd_str(select name from instruments)
FROM long_sequence(100);
```

throws `STRING constant expected`

**Comment by bluestreak01** [comment]
Am I right to assume this was attempted as a workaround for #92?  If so I would rather fix the root issue and leave this to be implemented if there is enough enthusiasm for it

**Comment by TheTanc** [comment]
Actually its the contrary, #90 was attempted as a workaround for #92. It worked well until I tried with special characters.

**Comment by bluestreak01** [comment]
Ok, understood. That said it would be much wider reaching if we fixed #90. It would perhaps make this issue redundant too?

**Comment by TheTanc** [comment]
Sure. will close.
